CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
+### Added
+- `ConvivaAnalyticsIntegration.ssai` namespace to enable server side ad tracking
```

Comment:
Did you intentionally go for a minimal entry here?
For iOS I listed every API with a very short description.

I chose a more verbose approach to the changelog entry but realized just now that I missed to update the readme which in your case compliments this, so this is more of a general question.

Reply:
I felt like the functions names are quite self explanatory, hence I did not put them here. But no strong opinion, happy to align with you changelog for consistency.

Reply:
My goal was to showcase, so if you only look at changelog it doesn't look much with a one-liner but also no strong opinion.

Reply:
Let's go with this simple one-liner as it feels heavy with the README together when having both verbose!